bindings/wallet-cordova/www/wallet.js

```diff
@@ -404,6 +449,18 @@ var plugin = {
         exec(successCallback, errorCallback, NATIVE_CLASS_NAME, FRAGMENT_ID, [transaction.buffer]);
     },
 
+    /**
+     * @param {string} settingsPtr
+     * @param {number} unixEpoch
```

Comment:
Should clarify, here and in other comment, that this parameter means seconds since Unix epoch.